onnxruntime/core/providers/qnn/builder/qnn_node_group/lpbqgemm_fusion.cc

```diff
@@ -178,23 +209,26 @@ Status UnpackWeightTensorData(const QnnModelWrapper& qnn_model_wrapper,
 }
 
 Status CreateOrValidateOnQnn(QnnModelWrapper& qnn_model_wrapper,
-                             const NodeUnit& scale_dql_node_unit,
-                             const NodeUnit& w_ql_node_unit,
-                             const NodeUnit& w_dql_node_unit,
-                             const NodeUnit& act_dql_node_unit,
-                             const NodeUnit& gemm_node_unit,
-                             const NodeUnit& output_ql_node_unit,
+                             const NodeUnit* p_scale_dql_node_unit,
+                             const NodeUnit* p_w_ql_node_unit,
+                             const NodeUnit* p_w_dql_node_unit,
+                             const NodeUnit* p_act_dql_node_unit,
+                             const NodeUnit* p_gemm_node_unit,
+                             const NodeUnit* p_output_ql_node_unit,
                              bool validate) {
+  const NodeUnit& scale_dql_node_unit = *p_scale_dql_node_unit;
+  const NodeUnit& w_dql_node_unit = *p_w_dql_node_unit;
+  const NodeUnit& act_dql_node_unit = *p_act_dql_node_unit;
+  const NodeUnit& gemm_node_unit = *p_gemm_node_unit;
+  const NodeUnit& output_ql_node_unit = *p_output_ql_node_unit;
```

Comment:
why update the parameters to be pointers if they will immediately be unconditionally dereferenced? now there is the possibility of passing in a nullptr by mistake.